### ISSUE TITLE
test REDS dataset in basicvsr

### ISSUE THREAD

**Issue by xuboming8** [open-mmlab/mmediting#483]
Limited by CUDA memory, we can't test with 100 frames in REDS.
For example,If I want to use 50 frames to test，I just change the parameter in config file"num_input_frames=100" to "num_input_frames=50".
Does the code automatically cut 100 frames into two 50 frames.Or we have to cut frames manually and change the reds4 folder name in "val_partition = ['000', '011', '015', '020']"?

**Comment by ckkelvinchan** [comment]
Using num_input_frames=50 will result in randomly selecting a starting frame and its following 50 frames. 

One easier way is to modify the forward function of basicvsr_net so that it process 50 frames each time, and concatenate them.

Another way is like what you mention. You can split them into different sequences.

**Comment by xuboming8** [comment]
Thanks！
I have another question about the division of training set and validation set of REDS.In the offical config file,The paths of training set, validation set and test set are consistent："lq_folder='data/REDS/train_sharp_bicubic/X4',gt_folder='data/REDS/train_sharp".It seems that this code is spliting REDS into training set(266 vedios) and validation set (REDS4) automatically.Is reds4 excluded from the training phase？Otherwise the results of REDS4 is not accurate.
Do we need to make sure the path of "data/REDS/train_sharp_bicubic/X4" only include 266 vedios except 270 vedios,and overwrite the path of validation set "data/REDS/val_sharp_bicubic/X4" only include 4 vedios (REDS4)?

**Comment by xuboming8** [comment]
> Using num_input_frames=50 will result in randomly selecting a starting frame and its following 49 frames.
> 
> One easier way is to modify the forward function of basicvsr_net so that it process 50 frames each time, and concatenate them.
> 
> Another way is like what you mention. You can split them into different sequences.

Thanks！
I have another question about the division of training set and validation set of REDS.In the offical config file,The paths of training set, validation set and test set are consistent："lq_folder='data/REDS/train_sharp_bicubic/X4',gt_folder='data/REDS/train_sharp".It seems that this code is spliting REDS into training set(266 vedios) and validation set (REDS4) automatically.Is reds4 excluded from the training phase？Otherwise the results of REDS4 is not accurate.
Do we need to make sure the path of "data/REDS/train_sharp_bicubic/X4" only include 266 vedios except 270 vedios,and overwrite the path of validation set "data/REDS/val_sharp_bicubic/X4" only include 4 vedios (REDS4)?

**Comment by ckkelvinchan** [comment]
You do not need to exclude them manually. The [dataset](https://github.com/open-mmlab/mmediting/blob/master/mmedit/datasets/sr_reds_multiple_gt_dataset.py#L50) has done it for you.

**Comment by xuboming8** [comment]
> You do not need to exclude them manually. The [dataset](https://github.com/open-mmlab/mmediting/blob/master/mmedit/datasets/sr_reds_multiple_gt_dataset.py#L50) has done it for you.

Thanks！！！
